lib/data/firebase_firestore/firebase_db_manager.dart

```diff
@@ -25,7 +25,7 @@ class FirebaseDbManager {
 
   static FirebaseDbManager get instance => FirebaseDbManager();
 
-  Future<String> AddBookImage(File? imageFile) async {
+  Future<String> uploadBookImage(File? imageFile) async {
     final timeStamp = DateTime.now();
 
     final fileRef = storageRef.child('${timeStamp.toString()}');
```

Comment:
Use book title as title for directory where you will place image.  The same directory that will be used for placing book page images.